### DISCUSSION TITLE
IPad Integrations?

### DISCUSSION THREAD

**Discussion by jmankoff** [remarkjs/remark#1361] [Q&A]
Hi all,

What are recommended best practices for using a remark slide deck to teach off an ipad -- is there a best practice for drawing on slides, viewing speaker notes, and so on?

**Comment by ChristianMurphy** [comment]
Welcome @jmankoff!
For slides you may be thinking of another project called remark, likely https://github.com/gnab/remark

Remark JS itself (this project) doesn't support slides, but there are wrapping projects like: https://github.com/jxnblk/mdx-deck, https://github.com/nkzw-tech/remdx and https://codehike.org/ which do build slide support on top of it.

**Reply by jmankoff** [reply]
Oh my bad, thank you!
